_posts/2017-07-30-nonbinary-tech.md

```diff
@@ -0,0 +1,23 @@
+
+**The Sound of Many Sleeping Bees** was a great talk on how phone notifications take us out of the presence of the moment, and how they can be like rude personal assistants who are constantly interrupting us.  They seem like a living thing, and by design we are encouraged to repeatedly interact with them.  I found myself comparing the annoying notifications of terrible apps to comments from people who haven't yet unlearned cissexism - each time it happens isn't bad enough to break any rules or code of conduct, but both are terribly draining.
+
+**Diversity by Default in Design** was given by a designer about their efforts to show diversity in their artwork, and the battles with companies to actually get that artwork used.  Eriol talked about how "edge-case" has become a word to discourage time being spent on making products accessible to marginalized groups and how making products friendly is universal, not just for the dominant user group.  Stock images for minority groups can throw up some worrying results, illiciting negative stereotypes, and it is important to remember that just because a stock image exists doesn't mean it is acceptable to use.  If you are in a position to represent your product users through photography, use photographs of actual users; and if those photographs don't show diversity then maybe you have bigger problems to address first.
+
+Damien Senger gave a very personal talk about **Building a New Life After Burnout**.  They reminded us the importance of saying no, taking breaks, and making deliberate choices to take care of ourselves.  Initial stages of burnout can show itself as us being less performant, poor memory, working more than the regular hours, fatigue, unstable mood, loneliness and exclusion, and risk behaviours.  It's important to take the time to do something about this as early as you notice this, even if it is just moving to another team.
+
+Olu shared a lot of great resources they have used through the beginning of their career in **Beyond 1s and 0s: inclusive spaces in binary places**.  [Diversity Tickets](https://diversitytickets.org) is a great place to find conferences that have ticket and travel grants for minority groups.  [WeAllJS](https://wealljs.org) is an inclusive Slack community for JavaScript developers.  The [LGBTQ in Technology](http://lgbtq.technology) Slack organisation is a space for LGBTQ people in technology to chat and support each other.  The [Women in Technology Chat](http://witchat.github.io) is a Slack organisation for women who work in technology.  Joining these communities helps with a sense of belonging, feeling welcomed and being understood which can be lacking elsewhere.  And remember to read and use the codes of conduct.
```

Comment:
I think the "And" in the last sentence is important, as it reinforces the idea that all of the previous things are good on their own but the codes of conduct which sit on top of them are so important.